### DISCUSSION TITLE
New pattern for mittens

### DISCUSSION THREAD

**Discussion by darigovresearch** [freesewing/freesewing#1617] [Forum]
**What is it that you would like to see happen?**

A pattern design for mittens. Mittens come in different variations, for an MVP (minimal viable product) having only one "pocket" for thumbs and one pocket for the remaining four fingers would suffice. 

https://en.wikipedia.org/wiki/Mitten

**Are you a FreeSewing patron?**

 - [ ] Yes, I am :hugs:
 - [x] No, I am not :thinking:

**Additional context**
This will likely be one of our first patterns to see what the contribution process is like & how to generate parametric patterns. Anybody who is new to contributing or interested in helping is welcome to join. See this video if you need some guidance on how you may be able to help - https://www.youtube.com/watch?v=UWA4wyacY2A

Suggested name is Mickey the mittens but suggestions are welcome

**Comment by starfetch** [comment]
Mittens would be really great to have! There has been some talk in the '#pattern-design' channel over at our Discord about a pattern for gloves by @/rostic, maybe there are some resources that could be pooled?

**Comment by darigovresearch** [comment]
@starfetch sounds great thanks for letting us know, getting everyone interested on the contributor's call today to discuss would great and make it easier to coordinate

https://github.com/freesewing/freesewing/issues/1580

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions

**Comment by darigovresearch** [comment]
Reopening discussion from the community call https://github.com/freesewing/freesewing/discussions/5532

**Comment by BenJamesBen** [comment]
Closing because there does not seem to be any interest for active discussion about this topic, per the April 27, 2024 Community Call. https://github.com/freesewing/freesewing/discussions/5724#discussioncomment-9247197

(Please feel welcome to re-open this Discussion if there is interest in resuming active discussion about the topic.)
